features/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/internal/SessionReplayFeatureTest.kt

```diff
@@ -323,20 +323,6 @@ internal class SessionReplayFeatureTest {
         verifyNoInteractions(mockRecorder)
     }
 
-    @Test
-    fun `M log warning and do nothing W startRecording() { feature is not initialized }`() {
-        // When
-        testedFeature.startRecording()
-
-        // Then
-        mockInternalLogger.verifyLog(
-            InternalLogger.Level.WARN,
-            InternalLogger.Target.USER,
-            SessionReplayFeature.CANNOT_START_RECORDING_NOT_INITIALIZED
-        )
-        verifyNoInteractions(mockRecorder)
-    }
-
```

Comment:
Now that the `startRecording` function does not perform the check, this test does not make sense anymore. 
It is replaced by the other tests below.